kuzzleio/kuzzle-device-manager
# feat(deviceManagerEngine): free devices after engine deletion

## What does this PR do ?
Unassign all the devices from their engineId and assetId in the admin index when the deviceManagerEngine is deleted **after** the deletion of its collections.

### How should this be manually tested?

  - Step 1 : Create a device-manager-engine
  - Step 2 : Add attach several devices to it and assign them to some assets
  - Step 3 : Delete the engine
  - Step 4 : Check in the admin index in the device collection that all former devices don't have any engineId or assetId anymore.

KZLPRD-511
